src/DetermineRoot.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\General;
+
+use Robo\Result;
+use Robo\Task\BaseTask;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * Determines the root of a Drupal project and adds it to the config.
+ * Config is available for classes implementing
+ * \Robo\Contract\ConfigAwareInterface and using \Robo\Common\ConfigAwareTrait.
+ *
+ * ``` php
+ * $this->taskDetermineRoot()
+ *     ->dir(getcwd())
+ *     ->run();
+ * $this->getConfig()->get('digipolis.root.project');
+ * ```
+ *
+ */
+class DetermineRoot extends BaseTask
```

Comment:
The key is called digipolis.root.project, so should we rename this class to DetermineProjectRoot?